met-web/src/components/engagement/form/EngagementFormTabs/EngagementSettings.tsx

```diff
@@ -229,6 +289,24 @@ const EngagementSettings = () => {
                         </Tooltip>
                     </ClickAwayListener>
                 </Grid>
+                <Grid item xs={12}>
+                    <PrimaryButton
+                        data-testid="update-engagement-button"
+                        sx={{ marginRight: 1 }}
+                        onClick={() => handleUpdateEngagement()}
```

Comment:
do we have to update all the engagement? Can we just call the update metadate at this stage

Reply:
We can just update the metadata but if they go to the metadata tab after changing the engagement form tab it wont save all of their data.

Reply:
but they can still go back and press save form the form changes right?

Reply:
yes but its confusing for the user i think it would be best if we save the entire form

Reply:
I see your point, but I see the confusion in saving the entire form when just the settings save is pressed